### PR TITLE
v4: custom session sig setter

### PR DESCRIPTION
**Pull request by himanshuchawla009**

### CODE REVIEW

File: src/mpcCoreKit.ts [hunk 1]
```diff
@@ -478,6 +466,20 @@ export class Web3AuthMPCCoreKit implements ICoreKit {
     }
   }
 
+  public async _finalizeOauthLogin(loginResponse: TorusKey, userInfo: UserInfo, persistSessionSigs = true, importTssKey?: string): Promise<void> {
+    const postBoxKey = this._getPostBoxKey(loginResponse);
+    this.torusSp.postboxKey = new BN(postBoxKey, "hex");
+
+    this.updateState({
+      postBoxKey,
+      postboxKeyNodeIndexes: loginResponse.nodesData?.nodeIndexes,
```
Comment (ieow) [inline src/mpcCoreKit.ts:475]: do we need to take nodeIndexes in to consideration?

File: package/package.json [hunk 2]
```diff
@@ -0,0 +1,117 @@
+{
```
Comment (ieow) [inline package/package.json:1]: Is this package folder's files needed?
is it created by error?

Comment (himanshuchawla009) [reply package/package.json:1]: ah, yeah its by mistake, i will remove it

File: src/mpcCoreKit.ts (hunk 1 above)
Comment (himanshuchawla009) [reply src/mpcCoreKit.ts:482]: yes, i need it so that passkey link api should know which nodes contains shares of the postbox key

**Review by ieow — APPROVED** [review]
Remove the package directory, else 
lgtm

**Comment by matthiasgeihs** [comment]
eh guys @himanshuchawla009 @ieow 
you merged the `dist` folder here.
this is just a build folder that shouldn't be part of the project source.

**Pull request merged**